src/ResponseModel/Cms/EventResponse.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace BalticRobo\Api\ResponseModel\Cms;
+
+use BalticRobo\Api\Model\Cms\Event\EventDTO;
+use BalticRobo\Api\ResponseModel\ResponseInterface;
+
+final class EventResponse implements ResponseInterface
+{
+    private $event;
+
+    public function __construct(EventDTO $event)
+    {
+        $this->event = $event;
+    }
+
+    public function respond(): array
+    {
+        $event = $this->event;
```

Comment:
This assign is not necessary.

Also don't add not necessary `event` nesting in return array. You are do request for Event so you know that response will be event info.

Also remember to use **sneak_case** in API request/response.

You can write:
```php
return [
  'data' => [
    'year' => $this->event->getYear(),
    'starts_at' => $this->event->getEventStartsAt()->format(\DateTime::DATE_ATOM),
    'registration' => [
      'starts_at' => $this->event->getRegistrationStartsAt()->format(\DateTime::DATE_ATOM),
      'stops_at' => $this->event->getRegistrationStopsAt()->format(\DateTime::DATE_ATOM),
    ],
  ],
];
```